3529/2/telegram.rb

```diff
@@ -0,0 +1,30 @@
+require 'telegram/bot'
+require 'yaml'
+require 'httparty'
+require 'open-uri'
+require_relative 'start_context'
+require_relative 'checkin_context'
+require_relative 'checkout_context'
+
+class WebhooksController < Telegram::Bot::UpdatesController
+  include StartContext
```

Comment:
Why do I need to include all the staff here?

Reply:
for creating message context?)

Reply:
I'm talking about includes all these modules when almost all of them include each other
Mb good idea is creating come `SomeStaffHelper` that contains all of one kind modules and inlcude it everywhere.

Reply:
okey, sounds better, then i had)